media-sound/audacity/audacity-3.2.2.ebuild

```diff
@@ -0,0 +1,221 @@
+# CC-BY-3.0 - Documentation
+LICENSE="GPL-2+
+	GPL-3
+	doc? ( CC-BY-3.0 )
+	audiocom? ( ZLIB )
+"
+SLOT="0"
+KEYWORDS="~amd64"
+IUSE="alsa audiocom doc ffmpeg +flac id3tag +ladspa +lv2 mad mpg123 ogg
+	opus +portmixer sbsms twolame vamp +vorbis wavpack"
+
+# The testsuite consists of two tests, 50% of which fail.
+RESTRICT="test"
+
+# dev-db/sqlite:3 hard dependency.
+# dev-libs/glib:2, x11-libs/gtk+:3 hard dependency, from
+#   cmake-proxies/cmake-modules/dependencies/wxwidgets.cmake
+# sys-apps/util-linux hard dependency, from cmake-proxies/CMakeLists.txt
+#   for libuuid
+# portmidi became non-optional: building without it results in build
+#   failures, even with some of the Debian patches.  It's probably not
+#   in our best interest to fix that as a patch series.
+# glib, gtk and gdk are all directly relied on in the source, not just
+
+# Libraries used at runtime via dlopen:
+# - dev-libs/{serd,sord} - for LV2 support
+# - media-libs/{opus,sratom} :: For Opus and LV2 respectively
+# - media-sound/lame :: For MP3 export
+# - media-video/ffmpeg :: For generic FFMPEG export
+#   This one has the interesting property of many versions being
+#   supported at runtime.  See: libraries/lib-ffmpeg-support/impl
+#   Current support grid:
+#   - Lavf - 5[789]
+#   - Lavc - 5[789]
+#   - Lavu - 5[2567]
+
+RDEPEND="dev-db/sqlite:3
+	dev-libs/expat
+	dev-libs/glib:2
+	media-libs/libsndfile
+	media-libs/libsoundtouch:=
+	media-libs/portaudio[alsa?]
+	media-libs/portmidi
+	media-libs/portsmf:=
+	media-libs/soxr
+	>=media-sound/lame-3.100-r3
+	x11-libs/gdk-pixbuf:2
+	x11-libs/gtk+:3
+	x11-libs/wxGTK:${WX_GTK_VER}[X]
+	sys-apps/util-linux
+	alsa? ( media-libs/alsa-lib )
+	audiocom? (
+		dev-libs/rapidjson
+		net-misc/curl
+	)
+	ffmpeg? ( media-video/ffmpeg )
+	flac? ( media-libs/flac:=[cxx] )
+	id3tag? ( media-libs/libid3tag:= )
+	lv2? (
+		dev-libs/serd
+		dev-libs/sord
+		>=media-libs/lilv-0.24.6-r2
+		media-libs/lv2
+		media-libs/sratom
+		media-libs/suil
+	)
+	mad? ( >=media-libs/libmad-0.15.1b )
+	mpg123? ( media-sound/mpg123 )
+	ogg? ( media-libs/libogg )
+	opus? ( media-libs/opus )
+	sbsms? ( media-libs/libsbsms )
+	twolame? ( media-sound/twolame )
+	vamp? ( media-libs/vamp-plugin-sdk )
+	vorbis? ( media-libs/libvorbis )
+	wavpack? ( media-sound/wavpack )
+"
+DEPEND="${RDEPEND}"
+BDEPEND="app-arch/unzip
+	sys-devel/gettext
+	virtual/pkgconfig
+"
+
+REQUIRED_USE="
+	mad? ( !mpg123 )
+"
+
+PATCHES=(
+	# Equivalent to previous versions
+	"${FILESDIR}/${PN}-3.2.2-disable-ccache.patch"
+	# From Debian
+	"${FILESDIR}/${PN}-3.2.2-fix-rpaths.patch"
+
+	# Disables some header-based detection
+	"${FILESDIR}/${PN}-3.2.2-allow-overriding-alsa-jack.patch"
+
+	# For has_networking
+	"${FILESDIR}/${PN}-3.2.2-remove-conan-threadpool.patch"
+	"${FILESDIR}/${PN}-3.2.2-allow-finding-rapidjson-pkgconfig.patch"
+)
+
+src_prepare() {
+	cmake_src_prepare
+
+	local header_subs="${S}/lib-src/header-substitutes"
+	cat <<-EOF >"${header_subs}/allegro.h" || die
```

Comment:
You used <<-EOF, so you could indent the bits below, but it's not needed to do so here if you don't want to.